geode-core/src/main/java/org/apache/geode/examples/security/ExampleSecurityManager.java

```diff
@@ -257,15 +257,65 @@ private void readUsers(final Map<String, User> rolesToUsers, final JsonNode node
     return roleMap;
   }
 
-  static class Role {
+  public static class Role {
     List<ResourcePermission> permissions = new ArrayList<>();
+
```

Comment:
Oh, I see.  Yes, because I made the object public, it's been a habit of mine to then make accessor methods for the protected fields.  I'll remove them.